### DISCUSSION TITLE
Have you tried Lamb?

### DISCUSSION THREAD

**Discussion by svandragt** [svandragt/lamb#79] [General]
I've not spoken to any Lamb users, but they might be out there. I'm enjoying using it for my own personal blog, but your use case mgiht be slightly different. 

I'd love to hear from you if you are still using or have stopped using Lamb and why!

**Comment by axodys** [comment]
I've had my eye on Lamb for a while, but haven't tried it yet because my existing solution has been adequate and I prefer the dark mode theme I created for it to the Lamb default. But I really like what you've done so far- the page, search, and tag support is quite cool and superior to what I've working with. Recently I've become partial to opening my posts with specific emoji to denote topic categories, but I noticed that the lamb search path doesn't seem to support emoji-based URLs as it gets passed on to your old site's search function.

**Reply by svandragt** [reply]
Hey axodys thanks for checking in :) That will be because I've never tested searches with emojis, but it's a cool idea. I've [opened an issue](https://github.com/svandragt/lamb/issues/80). I see the repo for your blog so will check that out as well. Very cool!

**Reply by svandragt** [reply]
You can now test this in the `main` branch if you like!
